srcpkgs/gprbuild/template

```diff
@@ -0,0 +1,49 @@
+# Template file for 'gprbuild'
+pkgname=gprbuild
+version=25.0.0
+revision=1
+build_style=gnu-configure
+short_desc="Advanced build system for multi-language programs"
+maintainer="Estevan Castilho <estevan.cps@gmail.com>"
+license=GPL-3.0
```

Comment:
These are going to be `GPL-3.0-or-later`.

Reply:
Had seen the COPYING3 file in the repository roots, but didn't pay too much attention to the file headers. I've also changed the libgpr and xmlada licenses to use `WITH GCC-exception-3.1`, as per https://spdx.org/licenses/GPL-3.0-with-GCC-exception.html.